arelle/Cntlr.py

```diff
@@ -18,15 +18,18 @@
 import subprocess
 import sys
 import tempfile
-from collections import defaultdict
-from typing import Any, Mapping, TYPE_CHECKING, TextIO, cast
+from typing import Any, TYPE_CHECKING, cast
 
 import regex as re
 
 from arelle import Locale, ModelManager, PackageManager, PluginManager, XbrlConst
 from arelle.BetaFeatures import BETA_FEATURES_AND_DESCRIPTIONS
 from arelle.SystemInfo import PlatformOS, getSystemWordSize, hasFileSystem, hasWebServer, isCGI, isGAE
 from arelle.WebCache import WebCache
+from arelle.logging.formatters import LogFormatter as _LogFormatter
+from arelle.logging.handlers.LogToBufferHandler import LogToBufferHandler
+from arelle.logging.handlers.LogToPrintHandler import LogToPrintHandler
+from arelle.logging.handlers.LogToXmlHandler import LogToXmlHandler
```

Comment:
Maybe reexport `LogHandlerWithXml` in case someone is importing it from this module to create their own handler?